docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -167,15 +141,14 @@ spec:
       pure: registry.connect.redhat.com/purestorage/openstack-manila-share-pure-rhosp-18-0
 ----
 +
-The name of the `OpenStackVersion` must match the name of your `OpenStackControlPlane`, so in your case it may be other than `openstack`.
-
-. If providing sensitive information, such as passwords, hostnames and
-usernames, it is recommended to use {OpenShift} secrets, and the
-`customServiceConfigSecrets` key. An example:
+The name of the `OpenStackVersion` CR must match the name of your `OpenStackControlPlane` CR.
+// - TODO link config diff tables for RHOSP 17.1 (Wallaby) to RHOSP 18 (Antelope) -
```

Comment:
Can we remove the “TODO” line?

Reply:
/me doesn't remember either

We can drop this until we know if there are any config options exist that impact upgrades (I don't think there are any besides what we spoke about in the Ceph-NFS sections)

Reply:
It doesn't hurt to leave the "TODO" line if we think we need it as a reminder. It's just nice to clean up these comments if they're not necessary anymore.